### PR TITLE
Fix it fix it fix it

### PR DESCRIPTION
**Pull request by TomasTomecek**
Fix it!

@rpitonak can you review please?

### CODE REVIEW

File: tests/config.yaml [hunk 1]
```diff
@@ -1,12 +1,12 @@
 document: modularity-testing
 version: 1
 name: docker.io/modularitycontainers/mariadb
-modulemd-url: http://pkgs.fedoraproject.org/cgit/modules/mariadb.git/plain/mariadb.yaml?h=f26
+modulemd-url: https://src.fedoraproject.org/modules/mariadb/raw/master/f/mariadb.yaml
 service:
     port: 3036
 module:
     docker:
-        start: "docker run -it --name mariadb_database -e MYSQL_USER=user -e MYSQL_PASSWORD=pass -e MYSQL_DATABASE=db -p 3306:3306"
```
Comment (rpitonak) [inline tests/config.yaml:9]: Tests start failing because of this change. In the [SQL queries test](https://github.com/container-images/mariadb/blob/master/tests/sqlqueries.py) the DB is referenced by this name to find out IP adress of the container.

**Review by rpitonak — CHANGES_REQUESTED** [review]

File: tests/config.yaml (hunk 1 above)
Comment (TomasTomecek) [reply tests/config.yaml:9]: tests were failing for me b/c MTF didn't not remove the containers during testing and docker complained about "container with such name already exists", let me try again

**Comment by TomasTomecek** [comment]
updated, tests finally passing

**Review by rpitonak — APPROVED** [review]
\o/
